test/test_log.py

```diff
@@ -27,18 +28,25 @@ def setUp(self):
         self.configure_mocked_logging('project-id', 'logger_name', 'workflow-id')
         self.test_logger = logging.getLogger('any.random.logger.name')
         self.root_logger = logging.getLogger('')
+
+    def _clear_all_root_loggers(self):
+        for h in logging.getLogger().handlers[:]:
+            logging.getLogger().removeHandler(h)
+            h.close()
  
     def tearDown(self):
-        logging.getLogger().handlers.clear()
+        self._clear_all_root_loggers()
         bigflow.log._LOGGING_CONFIGURED = False
 
     def test_should_create_correct_logging_link(self):
-        with self.assertLogs(level='INFO') as logs:
+
+        f = io.StringIO()
+        with contextlib.redirect_stderr(f):
```

Comment:
Why stderr?

Reply:
Because by default 'logging.basicConfigure()' just creates 'logging.StreamHandler(None)' and attaches it to the root logger.  StreamHandler output is 'sys.stderr' by default.